collectors/ioping.plugin/README.md

```diff
@@ -0,0 +1,80 @@
+# ioping.plugin
+
+The ioping plugin supports monitoring latency for any number of directories/files/devices,
+by pinging them with `ioping`.
+
+A recent version of `ioping` is required (one that supports option ` -N `).
+The supplied plugin can install it, by running:
+
+```sh
+/usr/libexec/netdata/plugins.d/ioping.plugin install
+```
+
+The above will download, build and install the right version as `/usr/local/bin/ioping`.
+
+Then you need to edit `/etc/netdata/ioping.conf` (to edit it on your system run
+`/etc/netdata/edit-config ioping.conf`) like this:
+
+```sh
+# uncomment the following line - it should already be there
+ioping="/usr/local/bin/ioping"
+
+# set here the directory/file/device, you need to ping
+destination="destination"
+
+# override the chart update frequency - the default is inherited from netdata
+update_every="1s"
+
+# the request size in bytes to ping the destination
+request_size="4k"
+
+# other iping options - these are the defaults
+ioping_opts="-T 1000000 -R"
+```
+
+## alarms
+
+netdata will automatically attach a few alarms for each host.
+Check the [latest versions of the ioping alarms](../../health/health.d/ioping.conf)
+
+## Multiple ioping Plugins With Different Settings
+
+You may need to run multiple ioping plugins with different settings or different end points.
+For example, you may need to ping one destination once per 10 seconds, and another once per second.
+
+netdata allows you to add as many `ioping` plugins as you like.
+
+Follow this procedure:
+
+**1. Create New ioping Configuration File**
+
+
+```sh
+# Step Into Configuration Directory
+cd /etc/netdata
+
+# Copy Original ioping Configuration File To New Configuration File
+cp ioping.conf ioping2.conf
+```
+
+Edit `ioping2.conf` and set the settings and the destination you need for the seconds instance.
```

Comment:
Aren't we using edit-config for this?

Reply:
sorry, mislead with the title, so this is to configure a second instance. Maybe adjust the title a bit?

Reply:
It is already in the section "Multiple ioping Plugins With Different Settings"